### DISCUSSION TITLE
Locator

### DISCUSSION THREAD

**Discussion by Spiderkb** [magicbug/Cloudlog#1474] [General]
I new install my server cloudlog.
Problem create new QSO. The entry of all locators refuses me.
Look picture
![Screenshot_20220508-150406](https://user-images.githubusercontent.com/105158721/167298205-68762ce5-dff2-483c-a1af-5c6df903fa68.jpg)

**Comment by magicbug** [comment]
I'm wondering if this is not locator related, have you created a station location? and also try updating the country files via Admin

**Reply by Spiderkb** [reply]
yes. Create new station location. How can I check if the database has been refreshed? Because it only says that download files and nothing changes.

**Reply by magicbug** [reply]
If it works correctly it will say "done" if that isn't happening that needs resolved and will be folder permissions see the wiki for info

**Reply by Spiderkb** [reply]
Which folder?

**Reply by Spiderkb** [reply]
Testing backup..... Working....

**Reply by Spiderkb** [reply]
I find problem
DEBUG - 2022-05-08 16:16:47 --> Migration class already loaded. Second attempt ignored.
ERROR - 2022-05-08 16:16:47 --> Severity: Warning --> gzopen(): https:// wrapper is disabled in the server configuration by allow_url_fopen=0 /home/eslone/s50kc.e-slo.net/Cloudlog/application/controllers/Update.php 166
ERROR - 2022-05-08 16:16:47 --> Severity: Warning --> gzopen(https://cdn.clublog.org/cty.php?api=a11c3235cd74b88212ce726857056939d52372bd): failed to open stream: no suitable wrapper could be found /home/eslone/s50kc.e-slo.net/Cloudlog/application/controllers/Update.php 166
ERROR - 2022-05-08 16:16:47 --> Severity: Warning --> gzeof() expects parameter 1 to be resource, boolean given /home/eslone/s50kc.e-slo.net/Cloudlog/application/controllers/Update.php 168
ERROR - 2022-05-08 16:16:47 --> Severity: Warning --> gzgetc() expects parameter 1 to be resource, boolean given /home/eslone/s50kc.e-slo.net/Cloudlog/application/controllers/Update.php 169

Enable allow_url_fopen. It working

**Reply by Spiderkb** [reply]
Now there is also an error in the call sign.
I also can't enter the locator.

https://snipboard.io/MyS23i.jpg

**Comment by Spiderkb** [comment]
I finally managed to make it work.
